### PR TITLE
Removed dependency of vault gen/passwd plugin

### LINKED ISSUE #44: Remove dependency of using vault gen/passwd plugin to generate secrets
Generating random string using Password lookup plugin
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/password_lookup.html

### PR DESCRIPTION
**Pull request by Neha-Sinha2305**
closes #44 
Generating random string using Password lookup plugin
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/password_lookup.html

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - Updated README.md #21
 - Added host volume for minio #20
 - Added tests to verify host volume for minio #34
-
+- Use ansible module to create random strings
```
Comment (zhenik) [inline CHANGELOG.md]:
```suggestion
- Use ansible module to create random strings #44
```

**Review by zhenik — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (Neha-Sinha2305) [reply CHANGELOG.md]: Thanks..updated it :+1:

**Comment by Neha-Sinha2305** [comment]
The test run in the latest build is not related to code changes, it is an intermittent issue seen when tests are run with github action.So, merging the code.

**Pull request merged**